chore: Add Demo Data sample (#90)
DataDog/dd-sdk-unity: samples/Demo Data/README.md

# Demo Data Project

The purpose of this project is to exercise E2E capabilities of the Datadog SDK, and produce

> Demoable data with some amount of variance.

Reviewer: ```suggestion
demo-able data with some amount of variance.
```

## Running

> The Datadog configuration is already setup, but is lacking the Client Token and Application ID. These
> can be populated by running `./add-keys.py`, which will use the `DATADOG_CLIENT_TOKEN` and
> `DATADOG_APPLICATION_ID` environment variables to populate the necessary keys in the settings file.

Reviewer: ```suggestion
The Datadog configuration is already setup, but is lacking the client token and application ID. These
can be populated by running `./add-keys.py`, which uses the `DATADOG_CLIENT_TOKEN` and
`DATADOG_APPLICATION_ID` environment variables to populate the necessary keys in the settings file.
```

> To run the application and send demo data to the environment, run `./run-demo.py`.

Reviewer 2: I don't see such file. Am I missing something?

Author: Nope, not done yet. I was documenting eventual state rather than current state. I'll remove it for now.

## Notes

To prevent committing changes to the `DatadogSettings.asset` file that might include a client token or

> RUM Application Id, consider running the following git command to ignore changes to the file:

Reviewer: ```suggestion
RUM Application ID, consider running the following git command to ignore changes to the file:
```
